guide/docs/faq/general.mdx

```diff
@@ -0,0 +1,272 @@
+# General
+
+:::note
+
+This is placeholder content that has been taken directly from the [documentation](https://docs.disnake.dev/en/latest/), and inherited from `discord.py`. This section will most likely be rewritten in the future.
+
+:::
+
+General questions regarding library usage belong here.
+
+### Where can I find usage examples?
+
+Example code can be found in the [examples folder](https://github.com/DisnakeDev/disnake/tree/master/examples) in the repository.
+
+### How do I set the "Playing" status?
+
+The `activity` keyword argument may be passed in the <DocsLink reference="disnake.Client" /> constructor or <DocsLink reference="disnake.Client.change_presence" />, given an <DocsLink reference="disnake.Activity" /> object.
+
+The constructor may be used for static activities, while <DocsLink reference="disnake.Client.change_presence" /> may be used to update the activity at runtime.
+
+:::warning
+
+It is highly discouraged to use <DocsLink reference="disnake.Client.change_presence" /> or API calls in <DocsLink reference="disnake.on_ready" /> as this event may be called many times while running, not just once. There is a high chance of disconnecting if presences are changed right after connecting.
+
+:::
+
+The status type (playing, listening, streaming, watching) can be set using the <DocsLink reference="disnake.ActivityType" /> enum.
+For memory optimisation purposes, some activities are offered in slimmed-down versions:
+
+-   <DocsLink reference="disnake.Game" />
+-   <DocsLink reference="disnake.Streaming" />
+
+Putting both of these pieces of info together, you get the following:
+
+```python
+client = disnake.Client(activity=disnake.Game(name="My game"))
+
+# Or, for watching:
+activity = disnake.Activity(
+    name="My activity",
+    type=disnake.ActivityType.watching,
+)
+client = disnake.Client(activity=activity)
+```
+
+### How do I send a message to a specific channel?
+
+You must fetch the channel directly and then call the appropriate method. Example:
+
+```python
+channel = client.get_channel(12324234183172)
+await channel.send("hello")
+```
```

Comment:
This example seems to use `get` instead of `fetch` (as stated above), which one do you think we should use?

Reply:
This is inherited from the docs themselves, so there's probably a proper reason behind using `get`. Should keep it the same, to be safe.